src/goto-instrument/goto_instrument_parse_options.cpp

```diff
@@ -1522,9 +1535,10 @@ void goto_instrument_parse_optionst::help()
     " --check-invariant function   instruments invariant checking function\n"
     " --remove-pointers            converts pointer arithmetic to base+offset expressions\n" // NOLINT(*)
     " --splice-call caller,callee  prepends a call to callee in the body of caller\n"  // NOLINT(*)
+    " --undefined-function-is-assume-false\n"
```

Comment:
Could we mark this as deprecated (possibly be removing it from help output, but leaving it in the .h file)? Also: is it now implemented via the new options? (Just being lazy, I should be able to figure out once I've completely read the code.)

Reply:
The effect can be replicated with `--generate-function-body '*' --generate-function-body-options assume-false`

Reply:
Yes indeed - but then we could get rid of the code implementing `undefined_function_abort_path`, couldn't we?

Reply:
@hannes-steffenhagen-diffblue I'd leave it to you whether to include such changes in this PR or instead just create an issue and make someone else (such as myself...) fix it.

Reply:
I've created #2070 to cover the additional cleanup and will merge this PR now.